ietf/group/models.py

```diff
@@ -442,8 +443,13 @@ def display_title(self):
             return self.title
         else:
             return self.artifact_type.name
-        
+
+    def is_markdown(self):
+        return self.content_type == "text/markdown;charset=utf-8"
```

Comment:
I'd be inclined to be general here and look only at the type (`text/markdown`) excluding the `charset`, but am not going to insist given the minimal likelihood that we are ever going to use that.

Reply:
I want to leave it this brittle edge for now - it will cause anything that is encoded differently (or that has a different content-type provided) to _not render_ as html, and I think that's what we'd prefer to have happen.